File: SDKs/Aspose.Words-Cloud-SDK-for-Java/src/main/java/com/aspose/words/model/SaveResponse.java
package com.aspose.words.model;

import com.aspose.words.model.SaveResult;
public class SaveResponse {
  private SaveResult SaveResult = null;
  private String Code = null;
  private String Status = null;
  /**
	 * getSaveResult
	 * Gets SaveResult
	 * @return SaveResult
	 */
  public SaveResult getSaveResult() {
    return SaveResult;
  }

	/**
	 * setSaveResult
	 * Sets SaveResult
	 * @param SaveResult SaveResult
	 */
  public void setSaveResult(SaveResult SaveResult) {
    this.SaveResult = SaveResult;
  }

  /**
	 * getCode
	 * Gets String
	 * @return Code
	 */
  public String getCode() {
    return Code;
  }

	/**
	 * setCode
	 * Sets String
	 * @param Code String
	 */
  public void setCode(String Code) {
    this.Code = Code;
  }

  /**
	 * getStatus
	 * Gets String
	 * @return Status
	 */
  public String getStatus() {
    return Status;
  }

	/**
	 * setStatus
	 * Sets String
	 * @param Status String
	 */
  public void setStatus(String Status) {
    this.Status = Status;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class SaveResponse {\n");
    sb.append("  SaveResult: ").append(SaveResult).append("\n");
    sb.append("  Code: ").append(Code).append("\n");
    sb.append("  Status: ").append(Status).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Words-Cloud-SDK-for-.NET/src/Com/Aspose/Words/Model/SaveResponse.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;

namespace Com.Aspose.Words.Model 
{
  /// <summary>
  /// 
  /// </summary>  
  public class SaveResponse : SaaSposeResponse 
  {                       
        /// <summary>
        /// Gets or sets SaveResult
        /// </summary>    
        public SaveResult SaveResult { get; set; }

        /// <summary>
        /// Get the string presentation of the object
        /// </summary>
        /// <returns>String presentation of the object</returns>
        public override string ToString()  
        {
          var sb = new StringBuilder();
          sb.Append("class SaveResponse {\n");
          sb.Append("  SaveResult: ").Append(SaveResult).Append("\n");
          sb.Append("}\n");
          return sb.ToString();
        }
    }
}
